src/xelor/message.py

```diff
@@ -46,48 +46,57 @@ def __repr__(self):
 class HDVMessage(NetworkReader):
     def __init__(self, data):
         super().__init__(data)
-        dest_folder = Path.home().joinpath(".xelor/data/Effects.json")
+        self.data = data
+        self.values = list()
+        items = list()
+
+        effect_file = Path.home().joinpath(".xelor/data/Effects.json")
         self.effect_dict = dict()
-        with open(dest_folder) as f:
+        with open(effect_file) as f:
             self.effect_dict = json.load(f)
 
-        dest_folder = Path.home().joinpath(".xelor/data/Items.json")
-        # with open(dest_folder) as f:
-        #    self.item = json.load(f)[context]
-        #    print(json.dumps(self.item))
+        # print(codecs.encode(self.data, "hex"))
 
-        self.data = data
-        _len, = struct.unpack_from("!h", self.data)
-        self.data = self.data[2:]
-        print("{} items found".format(_len))
-        for _ in range(_len):
-
-            uid = self.readVarShort()
-            _effect_len, = struct.unpack_from("!H", self.data)
-            self.data = self.data[2:]
+        self.read_int()  # Unknown data
+        _item_len = self.read_short()
+        print(_item_len)
```

Comment:
Instead of just printing the number of items found, I should ouput an explicit sentence.